src/sparseml/pytorch/sparsification/quantization/modifier_quantization.py

```diff
@@ -614,6 +617,59 @@ def _validate_params(self):
             )
             self._model_fuse_fn_name = "no_fuse"
 
+    def _log_quantization(
+        self,
+        module: Module,
+        epoch: float,
+        steps_per_epoch: int,
+    ):
```

Comment:
optional nit: if you remove the last comma for steps_per_epoch, it seems like it may be short enough to fit on one line?